framework/application/middlewares/responseHandler.ts

```diff
@@ -1,18 +1,27 @@
 import { NextFunction } from 'connect';
-import DbContext from '../../infrastructure/database/DbContext';
-import messages from '../../crosscutting/messages/message';
 import { ResponseRequest } from '../..';
 
+import logger from "../../crosscutting/logging/logger";
+import messages from '../../crosscutting/messages/message';
+import DbContext from '../../infrastructure/database/DbContext';
+
+
 export function responseHandler(req: any, res, next: NextFunction) {
   const json_ = res.json;
-  const DbContext = req.container.resolve('DbContext') as DbContext;
+  let DbContext;
+
+  try {
```

Comment:
instead of try/catch, I think we could use `if (req.container.has('DbContext)`

Ref: https://www.npmjs.com/package/awilix#containerhas

Reply:
Great i will make this change!

Reply:
Done!